spec/rest-middleware.spec.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+describe("restMiddleware", () => {
+    var configMock, HelmetMock, logger, restMiddleware, ServerMock, restifyLinks;
+
+    beforeEach(() => {
+        var LoggerMock, RestMiddleware;
+
+        RestMiddleware = require("../lib/rest-middleware");
+        LoggerMock = require("./mock/logger-mock");
+        HelmetMock = jasmine.createSpyObj("helmetMock", [
+            "frameguard",
+            "hidePoweredBy",
+            "hsts",
+            "ieNoOpen",
+            "noCache",
+            "noSniff",
+            "xssFilter"
+        ]);
+        ServerMock = jasmine.createSpyObj("serverMock", [
+            "use"
+        ]);
+        configMock = {
+            baseUrl: "http://localhost:8443",
+            https: false
+        };
+        restifyLinks = jasmine.createSpy();
+        logger = new LoggerMock();
+        restMiddleware = new RestMiddleware(HelmetMock, logger, restifyLinks);
+    });
+
+    it("calls restMiddleware without https", () => {
+        var next, req, res, standardLinks;
+        res = jasmine.createSpyObj("resMock", [
+            "links"
+        ]);
+        req = {
+            href: function () {
+                return "/path"
+            }
+        }
+        spyOn(req, "href").andCallThrough();
+        next = jasmine.createSpy("nextMock");
+        restMiddleware(configMock, ServerMock);
+        expect(logger.debug).toHaveBeenCalled();
```

Comment:
I really don't care if debug was called.  That's not part of the module's advertised functionality.